src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -28,20 +28,22 @@ public TorpedoStore(int numberOfTorpedos){
     }
   }
 
+  private static Random generator = new Random();
```

Comment:
The SonarCloud checks fail on the default random implementation, SecureRandom should be used instead.